.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!-- choose all that apply -->
+**Type of issue**: bug report | feature request | question | other enhancement
+
+<!-- if the issue is NOT a question, fill out the following sections -->
+
+<!-- choose one -->
+**Impact**: no rtl change | rtl refactoring | new rtl | unknown
+
+<!-- choose one -->
+**Development Phase**: request | proposal
+
+**Other information**
+<!-- include detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context -->
+
+<!-- if the current behavior is a bug, please provide the following steps to reproduce the problem -->
+
+**What is the current behavior?**
+
+**What is the expected behavior?**
+
+**Please tell us about your environment:**
+<!--    (examples)
+    - version: `chipyard git commit hash`
+    - os: `Linux knight 4.4.0-92-generic #115-Ubuntu SMP Thu Aug 10 09:04:33 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux`
+    - other: `prior steps taken/documentation followed/...`
+-->
+
+**What is the use case for changing the behavior?**
```

Comment:
I never liked this sentence; it doesn't really make sense to me. Maybe something like this would be clearer?
```suggestion
**What is a motivating example for changing the behavior?**
```